docs/configurations.md

```diff
@@ -1,53 +1,70 @@
 # ScalarDB Configurations
 
-This document describes the configurations for ScalarDB.
+This page describes the available configurations for ScalarDB.
 
-## Transaction manager configurations
+## ScalarDB client configurations
```

Comment:
I'm wondering if we really don't need to mention the ScalarDB Cluster things in this document.

@feeblefakie What do you think?

Reply:
@josh-wong I thought we decided to refer to the Cluster doc in the current ScalarDB Server section in a previous PR or somewhere.

Reply:
@feeblefakie @brfrn169 I'm sorry, I missed [this](https://github.com/scalar-labs/scalardb/pull/1082#discussion_r1322586289) when updating the docs for `master`, `3.10`, and `3.9`.

I've added a couple of sections about ScalarDB Cluster. Could you please take a look?:

- [Add `Use Consensus Commit through ScalarDB Cluster` section](https://github.com/scalar-labs/scalardb/pull/1190/commits/db16e34553d10d658014375f95b408bbdb883b0b)
- [Add `ScalarDB Cluster configurations` section](https://github.com/scalar-labs/scalardb/pull/1190/commits/d4977f215ca37d771968ba99f5b13932ffdbe175)

Reply:
@josh-wong Yes, that’s the PR. Thanks for the clarification!

Reply:
Thank you. I'll add these sections to the doc in `3.10` and `3.9`.